imports/node-app/core/createApolloServer.js

```diff
@@ -54,6 +56,12 @@ export default function createApolloServer(options = {}) {
 
       addCallMeteorMethod(context);
 
+      const localSchema = makeExecutableSchema({
+        typeDefs,
+        resolvers,
+        resolverValidationOptions
+      });
+      const fullSchema = mergeSchemas({ schemas: [localSchema, ...schemas] });
```

Comment:
Probably best to skip calling `mergeSchemas` if `schemas.length === 0`.

I would think we'd want to make both of these calls above the `expressServer.use` line, too. This function is called at the beginning of every single request that comes in, so it's only for things that need to be regenerated per-request.

Reply:
OK making these changes and testing now. Still WIP for unit tests.

Reply:
Updated code looks good